Wireframe.html

```diff
@@ -0,0 +1,34 @@
+<!DOCTYPE html>
+<html>
+<head>
+    <Title>Purpose of a Wireframe</Title>
+</head>
+<link rel="stylesheet" type="text/css" href="CSS1.css" />
+
+<body>
+    <header>
+        Wireframe
+    </header>
+    <div id="article1">

```

Comment:
Instead of `div`, you could use a semantic element here. Please, think which one. Same advise for other divs with id "article".